src/document/DocumentManager.ts

```diff
@@ -242,9 +242,20 @@ export class DocumentManager {
     });
     hotkeys.unbind();
     hotkeys("escape", () => {
+      this.model;
```

Comment:
This line seems unnecessary.

Reply:
not sure why that was even added..

Reply:
I don't see this on my branch actually..